analytics/java/src/main/java/com/koverse/examples/analytics/JavaWordCounter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Koverse, Inc.
+ * Copyright 2018 Koverse, Inc.
```

Comment:
Actually it should be 2019 now.

Reply:
Not sure why I updated that. I think we can leave the original dates in place